tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -25,6 +25,16 @@ spec:
       type: string
       description: Secret containing SSH private key for the Windows signing host
       default: windows-ssh-key
+    - name: macHostCredentials
```

Comment:
I don't see these used anywhere. Is it part of future work?

Reply:
Yes, CLDX-79 will have a PR for these very soon. https://github.com/konflux-ci/release-service-catalog/pull/566